colorbleed/scripts/fusion/deadline_switch_and_submit.py

```diff
@@ -0,0 +1,202 @@
+"""
+This module is for a standalone approach for Fusion similar to Maya.
+Note that this will require FusionConsoleNode.exe and the BlackmagicFusion
+module,
+
+Deadline runs a python process, lets call it P
+
+P will start the FusionConsoleNode in a new SUBPROCESS
+This SUBPROCESS will need to have the same environment as P to ensure it can
+use AVALON
+
+    P --> SUBPROCESS (FusionConsoleNode.EXE /listen)
+
+From the SUBPROCESS comes a Fusion Console Node which will be used as the Fusion
+instance to work in. In order to get the correct Fusion instance we use a
+ScriptServer to get all Fusion programs which are running.
+This is done by comparing the process ids with the subprocess.pid.
+
+    See `get_fusion_instance` function for more details
+
+In `avalon.fusion.pipeline` we have create a work around to get the fusion
+instance. This is done through
+
+    getattr(sys.module["__main__"], "fusion", None)
+
+Because we do this we can also allow to set correct fusion module, this is done
+by using the setattr. This will ensure that all other functions which are run
+within `process()` can find `fusion`.
+
+"""
+
+
+import subprocess
+import traceback
+import site
+import time
+import sys
+import os
+
+# This script only works with Python 2.7 and 3.6
+version = "{0}{1}".format(*sys.version_info)  # {major}{minor}
+assert version in ["27", "36"], "Script only works in Python 2.7 or 3.6"
+key = "FUSION_PYTHON{0}_HOME".format(version)
+
+# Set Python 3.6 home for fusion, debug
```

Comment:
This isn't for debugging purposes but is a explicit and direct fix for a crash on `BlackmagicFusion` library crashing.

Let's state that in the comment so the intent is clear.

This could just read:
```python
# Importing BlackmagicFusion package in standalone Python interpreter
# crashes when not installed on default location but runs from, e.g. a
# network share. Forcing Fusion's Python home magically fixes it.
print("Workaround BlackmagicFusion import crash by setting %s" % key)
os.environ[key] = os.path.dirname(sys.executable)
```